userguide/kaui/Kaui_Guide_Draft.adoc

```diff
@@ -933,31 +933,228 @@ To delete a payment method:
+
+==== Add a Subscription
+
+[NOTE]
+*Note:* To create a subscription, you must have at least one product and one plan defined in the catalog. For information on adding plans to the catalog, see <<part_6_tenants_and_admin>>.
+
+1. On the Subscriptions page, click the plus sign to the right of *Subscription Bundles:*
+
+image::SelectAddSubscription.png[width=350,align="center"]
+
+Kaui displays the Add New Subscription screen:
+
+image::AddNewSubscription.png[width=450,align="center"]
+
+[start=2]
+. Fill in the fields. For field information, see <<Add Subscription Field Descriptions>>.
+. Click the *Save* button.
+
+==== Add Subscription Field Descriptions
+
+[cols="1,3"]
+|===
+^|Field ^|Description
+
+| Bundle Key
+| If you leave this field blank, Kill Bill generates a unique bundle key. If necessary, you can enter a different bundle key.
+
+|Subscription Key
+| If you leave this field blank, Kill Bill generates a unique subscription key. If necessary, you can enter a different subscription key.
+
+| Plan
+| Select the plan from the dropdown list. These options come from the plans defined in the catalog.
+
+| Price Override
+| If you need to override the price for the plan you've selected, enter an amount in this field.
+
+| Immediate Creation
+| Select this option to start the subscription immediately.
+
+| Specify a date
+| To define a specific date on which the subscription begins, select this option, and choose a date from the *Date* field that appears.
+
+|===
+
+==== Add an Add-On Product to a Subscription
+
+This section explains how to add an add-on product to a customer's subscription.
+
+[NOTE]
+*Note:* To add an add-on to a subscription, the add-on must exist in the catalog. For information on adding an add-on to the catalog, see <<part_6_tenants_and_admin>>.
+
+1. Open the account and click *Subscriptions* on the sub-menu.
+2. Hover over the gift icon and click *Add add-on*:
+
+image::Subscriptions-SelectAddOnIcon.png[width=350,align="center"]
+
+Kaui opens the Add New Add-On screen:
+
+image::AddNewAddOn.png[width=450,align="center"]
+
+[start=3]
+. Fill in the fields. For field information, see <<Add New Add-On Field Descriptions>>.
+
+==== Add New Add-On Field Descriptions
+
+[cols="1,3"]
+|===
+^|Field ^|Description
+
+|Subscription Key
+| Note that this subscription key applies to the add-on, not the original subscription. If you leave this field blank, Kill Bill generates a unique subscription key. If necessary, you can enter a different subscription key.
+
+| Plan
+| Select the add-on to add to the subscription.
+
+| Price Override
+| If you need to override the price for the add-on you've selected, enter an amount in this field.
+
+| Immediate Creation
+| Select this option to start the subscription immediately.
+
+| Specify a date
+| To define a specific date on which the subscription begins, select this option, and choose a date from the *Date* field that appears.
+
+|===
+

```

Comment:
Below the "Add Add-on", there are other options like "Transfer Ownership" and "Pause/Resume". Are these intentionally left out?

Reply:
See the ALL NEW Pause/Resume section below (1040). Pierre said to not document the Transfer Ownership feature for now.